lib/internals/routes.js

```diff
@@ -32,6 +32,11 @@ module.exports = (
       error_description: errorDescription
     } = authorisationErr
 
+    let state
+    if (authorisationErr && authorisationErr.state) {
+      state = JSON.parse(authorisationErr.state)
+    }
+
```

Comment:
state can be undefined at this point, and is later passed as a parameter - is this intentional?

Reply:
Set a default value and check before adding a query key